libraries/exoplayer_rtsp/src/main/java/androidx/media3/exoplayer/rtsp/RtspMediaTrack.java

```diff
@@ -208,6 +211,21 @@ public int hashCode() {
       case MimeTypes.AUDIO_AAC:
         checkArgument(channelCount != C.INDEX_UNSET);
         checkArgument(!fmtpParameters.isEmpty());
+        if(mediaEncoding.equals(RtpPayloadFormat.RTP_MEDIA_MPEG4_AUDIO)) {
+          boolean isConfigPresent  = true;
+          if (fmtpParameters.get(PARAMETER_MP4A_C_PRESENT) != null && fmtpParameters.get(
+              PARAMETER_MP4A_C_PRESENT).equals("0")) {
+            isConfigPresent  = false;
+          }
+          checkArgument(!isConfigPresent, "cpresent == 0 means we need to parse config");
+          @Nullable String configInput = fmtpParameters.get(PARAMETER_MP4V_CONFIG);
+          if (configInput != null && configInput.length() % 2 == 0) {
```

Comment:
since we require `cpresent=0`, then fmtp must include `config` right? can we assert on `configInput != null && configInput.length() % 2 == 0`? 

I'm not sure if this requirement applies to all ADTS/LATM streams.